Dockerfile

```diff
@@ -12,20 +12,31 @@
 # Build step: $ docker build -t eclipse-che-dashboard .
 # It builds an archive file that can be used by doing later
 #  $ docker run --rm eclipse-che-dashboard | tar -C target/ -zxf -
+
 FROM node:8.16.0
 
 RUN apt-get update \
-    && apt-get install -y git \
+    && apt-get install -y git curl \
     && apt-get -y clean \
     && rm -rf /var/lib/apt/lists/*
 
+RUN if [ "$(uname -m)" = "ppc64le" ]; then \
+     mkdir /tmp/phantomjs \
+     && curl -Ls "https://github.com/ibmsoe/phantomjs/releases/download/2.1.1/phantomjs-2.1.1-linux-ppc64.tar.bz2" | tar -xj --strip-components=1 -C /tmp/phantomjs \
+     && cd /tmp/phantomjs \
+     && mv bin/phantomjs /usr/local/bin \
+     && rm -rf /tmp/phantomjs; fi
+
 COPY package.json /dashboard/
 COPY yarn.lock /dashboard/
 WORKDIR /dashboard
 RUN yarn install --ignore-optional
```

Comment:
@bivasda1 I think 
```
RUN yarn install --ignore-optional 
```
shouldn't be there in the case with 
```
RUN if [ "$(uname -m)" = "ppc64le" ]; then export QT_QPA_PLATFORM=offscreen; fi \
    && yarn install --ignore-optional
```

Reply:
WDYT?

Reply:
@atoulme Thanks for the suggestion.I have changed accordingly.

Reply:
@olexii4 I have made changes can u please check it once and let me know.